google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/models/ChangeStreamMutation.java

```diff
@@ -145,15 +160,27 @@ abstract static class Builder {
 
     abstract Builder setSourceClusterId(@Nonnull String sourceClusterId);
 
-    abstract Builder setCommitTimestamp(Instant commitTimestamp);
+    Builder setCommitTime(java.time.Instant commitTimestamp) {
+      return setCommitTimestamp(toThreetenInstant(commitTimestamp));
+    }
+
+    /** This method is obsolete. Use {@link #setCommitTime(java.time.Instant)} instead. */
+    @ObsoleteApi("Use setCommitTime(java.time.Instant) instead")
+    abstract Builder setCommitTimestamp(org.threeten.bp.Instant commitTimestamp);
 
     abstract Builder setTieBreaker(int tieBreaker);
 
     abstract ImmutableList.Builder<Entry> entriesBuilder();
 
     abstract Builder setToken(@Nonnull String token);
 
-    abstract Builder setEstimatedLowWatermark(Instant estimatedLowWatermark);
+    Builder setLowWatermarkTime(java.time.Instant estimatedLowWatermark) {
```

Comment:
Same question as for commit time. Also if we need to keep both setters this should be 'setEstimatedLowWatermarkTime'

Reply:
Yes, done.